src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Configuration/PlatformSettings.cs

```diff
@@ -7,20 +7,27 @@ namespace Altinn.Common.PEP.Configuration
     /// </summary>
     public class PlatformSettings
     {
+        private string _subscriptionKey;
+        private string _apiAuthorizationEndpoint;
+
         /// <summary>
         /// Gets or sets the url for the API Authorization endpoint
         /// </summary>
-        public string ApiAuthorizationEndpoint { get; set; }
+        public string ApiAuthorizationEndpoint
+        {
+            get => Environment.GetEnvironmentVariable("PlatformSettings__ApiAuthorizationEndpoint") ?? _apiAuthorizationEndpoint;
```

Comment:
GetEnvironmentVariable should be unnecessary if PlatformSettings is provided by the Configuration class in .Net Core. The only thing we must make sure of is that the different configuration sources is read in correct order. I believe this is the case for all existing platform and app application.